scout/dashboard.md

```diff
@@ -19,8 +19,9 @@ The **Overview** tab shows the total number of vulnerabilities across all your D
 
 ## Repository settings
 
-Enable Docker Scout analysis on repositories in the current organization by selecting your profile
-picture and then the **Repository settings** menu item.
+![A screenshot showing where to find the settings page for Docker Scout](./images/dashboard-overview-settings.png)
```

Comment:
Do we really need this screenshot here? I believe our users are smart enough to locate the settings icon by reading the description below.

Reply:
Docker Desktop has the same icon. It's called the "Settings" icon- https://docs.docker.com/desktop/settings/mac/ , which is generic enough for non-native speakers. Let's use the settings icon to be consistent.